docs/js-guide/amd-to-esm.rst

```diff
@@ -179,7 +165,9 @@ The final module dependency structure will look something like:
 Note that ``import "commcarehq";`` has been moved to the top of the file. The ordering is
 for consistency purposes, but it's important that either ``import "commcarehq";`` or
 ``import "commcarehq_b3";`` (for Bootstrap 3 / ``webpack_main_b3``) is present in the list
-of imports for Webpack Entry Point modules.
+of imports for Webpack Entry Point modules. If this import is not present in an entry point,
```

Comment:
This is interesting, it sounds like someone testing might not notice a missing `commcarehq`. I don't know that there's anything to do about it, it's good that it's documented.